### DISCUSSION TITLE
LaserGRBL: Program stays on "alarm", how to reset to "idle"?

### DISCUSSION THREAD

**Discussion by dennisdesab** [arkypita/LaserGRBL#1154] [Q&A]
Trying to set up LaserGRBL to Orthur Master 2 Laser Engraver.  The LaserGRBLR configuration indicates "ALARM", I need to change that to "IDLE" but do not know how to.  Ani suggestions?

**Comment by gmmanonymus111** [comment]
GRBL tells you what to do. 
`[MSG: '$H' | '$X' to unlock]`
So, either send `$X` to unlock or click the `Lock` button, or `$H` to home or click the `House` button (if enabled, at custom buttons and not at the movement arrows),
